LoRaEngine/deployment.template.json

```diff
@@ -63,15 +63,15 @@
             "type": "docker",
             "settings": {
               "image": "${MODULES.LoRaWanPktFwdModule.arm32v7}",
-              "createOptions": "  {\r\n  \"HostConfig\": {\r\n    \"NetworkMode\": \"host\", \"Privileged\": true \r\n\r\n  },\r\n  \"NetworkingConfig\": {\r\n    \"EndpointsConfig\": {\r\n      \"host\": {}\r\n    }\r\n  }\r\n}\r\n  \r\n  \r\n  \r\n"
+              "createOptions": "  {  \"HostConfig\": {    \"NetworkMode\": \"host\", \"Privileged\": true   },  \"NetworkingConfig\": {    \"EndpointsConfig\": {      \"host\": {}    }  }}      "
             },
             "env": {
               "RESET_PIN": {
-                "value": "7"
+                "value": "25"
```

Comment:
default here should be 7 like for seeedstudio. actually we should move it as .env variable